### DISCUSSION TITLE
Keys 'home' and 'end' should bring to start and end of the line when editing text.

### DISCUSSION THREAD

**Discussion by polina-c** [community/community#33854] [Pull Requests]
When I comment on issue or pr, I have habit to use the keys 'home' and 'end'. But, instead of bringing me to the start or end of the line, they bring me to the start or end of the page. Is it possible to give them different meaning while text editing, so that users do not have to break their typing habits?

I use Chrome. I understand it is Chrome's standard behavior.  Is it possible to override it?

**Comment by Simon-Ravitz** [comment]
Hey, @polina-c 
I'd suggest using templates for the PR and Issue in order to not change/edit the content manually.
I think it is the best practice 
Thanks

**Reply by airtower-luna** [reply]
A good template can *reduce* typing, but you still need to fill in the specifics of your issue/PR, and for comments (as in the question) they aren't applicable at all.

**Reply by polina-c** [reply]
I do not understand how template can help. Normally, every PR contains a lot of comments that need to be typed.
Did I misinterpret something?

**Reply by Simon-Ravitz** [reply]
@polina-c 
Hey, It is better to hold a file called  PR.yaml  inside the  workflows/GitHub folder inside your project. 
the file contains ready template for PR, and for each PR you just gonna mark the neccecrey things you have decided, such:
Tests were passed?
Jira ticket link:
and more questions to ensure stuff

**Reply by Simon-Ravitz** [reply]
it's by Git actions

**Reply by polina-c** [reply]
I am still confused. There are tens of comments in this per, for example: https://github.com/flutter/devtools/pull/4501.
How are they going to be addressed by a template?

**Comment by airtower-luna** [comment]
I believe that is a browser issue. In Firefox it works for me like you expect (assuming the text area is active), so GitHub doesn't seem to be doing anything to break it. :sweat_smile: 

If you mention which browser you use maybe someone can help, or you could send a bug report to the browser.

**Reply by polina-c** [reply]
Thank you. Added the information to the question.
